.travis.yml

```diff
@@ -0,0 +1,32 @@
+env:
+  global:
+    - CC_TEST_REPORTER_ID=
```

Comment:
@szeck87 Are you able to get this ID?

Reply:
I need more access to this repo.

Reply:
@szeck87 try now :).

Reply:
b32edb620b73c1bd0e42e7a0157494c5841273f45cf1fae873375e018e6fa89d

Reply:
@muuvmuuv use the above ^.